src/tokens.scss

```
@@ -0,0 +1,6 @@
+@import "./../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables";
+
+:host {
+  --ds-auro-table-background-color: var(--ds-color-container-secondary-default, #{$ds-color-container-secondary-default});
```

I realized we need a more precise token name here, This is not the background color of the whole table. We may want to support that in the future but currently we only have a color for the alternating rows.

```suggestion
  --ds-auro-table-row-alt-container-color: var(--ds-color-container-secondary-default, #{$ds-color-container-secondary-default});
```